admin/context_test.go

```diff
@@ -0,0 +1,58 @@
+package admin_test
```

Comment:
we haven't decided where to put unit tests. an independent  package or follow go convention.